src/components/config_profile/test/profile_test.cc

```diff
@@ -724,6 +724,193 @@ TEST_F(ProfileTest, CheckIntContainerInSecurityData) {
+  // Check initial hmi_capabilities_file_name
+  EXPECT_EQ(profile_.app_config_folder() + "/" + "hmi_capabilities.json",
+            profile_.hmi_capabilities_file_name());
+}
+
+TEST_F(ProfileTest, CheckDefaultTimeout) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial default_timeout
+  EXPECT_EQ(10000u, profile_.default_timeout());
+}
+
+TEST_F(ProfileTest, CheckAppResumingTimeout) {
+  EXPECT_EQ(3000u, profile_.app_resuming_timeout());
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial app_resuming_timeout
+  EXPECT_EQ(3u, profile_.app_resuming_timeout());
+}
+
+TEST_F(ProfileTest, CheckAppResumptionSavePersistentDataTimeout) {
+  EXPECT_EQ(10000u, profile_.app_resumption_save_persistent_data_timeout());
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial app_resumption_save_persistent_data_timeout
+  EXPECT_EQ(10u, profile_.app_resumption_save_persistent_data_timeout());
+}
+
+TEST_F(ProfileTest, CheckVideoStreamingPort) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial video_streaming_port
+  EXPECT_EQ(5050u, profile_.video_streaming_port());
+}
+
+TEST_F(ProfileTest, CheckAudioStreamingPort) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial audio_streaming_port
+  EXPECT_EQ(5080u, profile_.audio_streaming_port());
+}
+
+TEST_F(ProfileTest, CheckStopStreamingTimeout) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial stop_streaming_timeout
+  EXPECT_EQ(1u, profile_.stop_streaming_timeout());
+}
+
+TEST_F(ProfileTest, CheckTimeTestingPort) {
+  EXPECT_EQ(5090u, profile_.time_testing_port());
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial time_testing_port
+  EXPECT_EQ(8090u, profile_.time_testing_port());
+}
+
+TEST_F(ProfileTest, CheckAppDirQuota) {
+  EXPECT_EQ(5090u, profile_.time_testing_port());
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial app_dir_quota
+  EXPECT_EQ(104857600u, profile_.app_dir_quota());
+}
+
+TEST_F(ProfileTest, CheckVideoServerType) {
+  EXPECT_TRUE(profile_.video_server_type().empty());
```

Comment:
@Ypostolov Where this value came from?

Reply:
@AByzhynar 
fixed in https://github.com/smartdevicelink/sdl_core/pull/1995/commits/8b04ba878df1f894f8688d2196f7a6c4b29b9328